Formula/o/oxipng.rb

```diff
@@ -20,8 +20,10 @@ class Oxipng < Formula
 
   def install
     system "cargo", "install", *std_cargo_args
+    cargo_run_args = std_cargo_args.reject { |arg| ["--root=#{prefix}", "--path=."].include?(arg) }
+    system "cargo", "run", "--manifest-path", "xtask/Cargo.toml", *cargo_run_args, "--", "mangen"
```

Comment:
why not just not use `std_cargo_args`

Reply:
To my knowledge, `--root` and `--path` are not valid arguments for `cargo run`; they are only valid arguments for `cargo install`.

Reply:
yeah, I am saying that just drop `std_cargo_args` for `cargo run`, it is a bit confusing.

Reply:
Thanks!